sections/semantics-forms.include

```diff
@@ -6336,6 +6336,33 @@ You cannot submit this form when the field is incorrect.</samp></pre>
 
   </div>
 
+<h6 id="the-autocapitalize-attribute">The <code>autocapitalize</code> attribute</h6>
+
+    The <dfn element-attr for="input"><code>autocapitalize</code></dfn> attribute provides
+    information to the user agent on the text capitalization a user's keyboard should
+    use on text fields.
+
+    It currently provides three different capitalization methods; <code>sentences</code>,
+    <code>words</code>, and <code>characters</code>.
```

Comment:
Please provide a clear normative definition of each mode. Also, what happens if the attribute value is not one of those?